### ISSUE TITLE
Add --roles flag to verify, lookup CLI commands

### ISSUE THREAD

**Issue by riyazdf** [docker/notary#857]
From discussion in https://github.com/docker/notary/issues/852#issuecomment-233820701

**Comment by HuKeping** [comment]
I can handle this but before that I also want to make sure this is the right way to do it. How about still discuss in #852

**Comment by riyazdf** [comment]
Sounds good, more than happy to discuss here as well if that thread gets hard to follow :)

**Comment by HuKeping** [comment]
@riyazdf  I'll pick this .
